task_processing/plugins/kubernetes/kubernetes_pod_executor.py

```diff
@@ -331,10 +339,22 @@ def __update_modified_pod(self, pod: V1Pod, event: Optional[PodEvent]) -> None:
             )
             return
 
-        logger.info(
-            f"Ignoring MODIFIED event for {pod_name} as it did not result "
-            "in a state transition",
-        )
+        if self.emit_events_without_state_transitions:
+            self.event_queue.put(
+                task_event(
+                    task_id=pod_name,
+                    terminal=False,
+                    timestamp=time.time(),
+                    raw=raw_event,
+                    task_config=task_metadata.task_config,
+                    platform_type="running",
```

Comment:
one other thing we could probably do in the future (if it'd be helpful) is expand the types of task_events we enqueue (or add additional data (or take over `raw` and put custom data in there))

but this is probably good enough for a while

Reply:
what other events might be of interest?

Reply:
what other events might be of interest?

Reply:
I was mostly just thinking we could make nice custom events for stuff like OOMKills like we had in Mesos (and maybe make a custom event for the main container exiting (but I guess that really bakes in the current hardcoded `main` container name))

fwiw, i don't think we should do this now: I was really just thinking out loud